### DISCUSSION TITLE
Why no Lollipop ?

### DISCUSSION THREAD

**Discussion by cyring** [community/community#76] [Mobile] [answered]
According to the Android Play Store, my LG G2 with a factory Lollipop ROM can't deserve GitHub Mobile. Why that ?

**Comment by aconsuegra** [comment, marked as answer]
Hey @cyring,

thanks for your feedback! Android Lollipop was released 6 years ago and it is not officially supported by Google anymore. In order to create a better experience for Android, we had to stop supporting this old version and focus on more modern versions of the OS. 

You can still experience a complete GitHub experience on your device by using our mobile website.
